### DISCUSSION TITLE
希望终端可以发送系统通知

### DISCUSSION THREAD

**Discussion by linuxlionking** [linuxdeepin/developer-center#3162] [Features Request & Ideas | 特性请求 & 头脑风暴]
比如，当终端在后台执行完成一个比较耗时的命令，发送一个执行完成的系统通知，是比较方便使用的。

**Comment by BLumia** [comment]
这个直接在你的脚本里执行就可以了呀。举个例子：

``` shell
$ sleep 5 && notify-send "五秒过去了"
```

**Reply by linuxlionking** [reply]
这样固然可以，其实希望将该功能内置到终端中。(看到GNOME 新终端程序实现了该功能 )

**Reply by BLumia** [reply]
GNOME 新终端程序实现了该功能是指 [这个第一方插件](https://github.com/gnome-terminator/terminator/pull/205) 吗？

**Reply by linuxlionking** [reply]
是的

**Reply by BLumia** [reply]
感觉可以考虑一下终端应该怎么加接口来支持这种形式的插件存在会合适一些。
